src/components/pages/advanced.tsx

```diff
@@ -43,6 +45,36 @@ function Advanced(props:AdvancedProps): JSX.Element {
     ],
   ];
 
+
+  const refList = [React.useRef<RewardElement>(null), React.useRef<RewardElement>(null),
+    React.useRef<RewardElement>(null), React.useRef<RewardElement>(null)];
+  const robotImages = [robot1, robot2, robot3, robot4];
+
+  function robots(index: number) {
+    switch(index) {
+      case 2:
+        refList[2].current?.rewardMe();
+        props.onCorrect();
+        break;
+      default:
+        refList[index].current?.punishMe();
+        break;
+    }
+  }
+
+  function getAltText(index: number) {
+    switch(index) {
+      case 0:
+        return 'red robot, circle head, with mouth, 1 antenna, 2 arms, 1 button';
+      case 1:
+        return 'red robot, square head, with mouth, 1 antenna, 0 arms, 1 button';
+      case 2:
+        return 'blue robot, triangle head, with mouth, 2 antenna, 2 arms, 3 buttons';
+      default:
+        return 'green robot, circle head, with mouth, 2 antenna, 2 arms, 1 button';
+    }
```

Comment:
Can we turn this into just an array of strings, it makes it more readable that way. This goes for all the other pages that has the getAltText function.